### PR TITLE
Add Hammond app.

### PR DESCRIPTION
**Pull request by alatiera**
There is not an Application Icon as of this moment.

The appdata.xml and .desktop file are distributed with the build system.

### CODE REVIEW

File: org.gnome.Hammond.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "build-options" : {
+        "env": {
+            "CARGO_HOME": "/run/build/Hammond/cargo"
+        },
+        "build-args": [ "--share=network" ]
```
Comment (TingPing) [inline org.gnome.Hammond.json]: You can't have network access, you need to do releases with all dependencies bundled (`cargo lock` I believe?).

**Comment by TingPing** [comment]
> There is not an Application Icon as of this moment.

I don't know if this should or shouldn't be a hard requirement but it dramatically makes your app less discoverable since as mentioned it won't show up in any graphical app store.

File: org.gnome.Hammond.json (hunk 1 above)
Comment (alatiera) [reply org.gnome.Hammond.json]: Oh yea, good thing that they already included in the tarball. 
It builds fine without network access. Pushing a fix in a bit.

**Comment by alatiera** [comment]
As for the Icon, As long it's possible to be added later I am fine with the first release not having one.

It's still a rough prototype and not ideal for users yet anyway.

**Comment by TingPing** [comment]
bot, build org.gnome.Hammond

**Comment by TingPing** [comment]
> It's still a rough prototype and not ideal for users yet anyway.

Then it does not belong on Flathub.

**Comment by alatiera** [comment]
By prototype I mean that it's design it's not well though and most of the parts will be re-written on the versions ahead once proper planning/designing is done.

I think the program as of now is in a usable state. I've written tests for all of the backend code and the gtk client won't segfault randomly. 

That said I would understand if it wouldn't yet be accepted.

**Comment by alexlarsson** [comment]
Yes, we don't want experimental or early stuff on flathub, that is not the point of it. We want stable, released, supported apps.

**Comment by alatiera** [comment]
Ok then, thanks for clearing it out. I guess can re-open the issue if/when there's a stable release.

**Pull request closed without merging**